src/index.js

```diff
@@ -38,6 +41,20 @@ export default async function loader(content, sourceMap, meta) {
 
   const postcssFactory = options.implementation || postcss;
 
+  // Check postcss versions to avoid using PostCSS 7
+  if (!isCheckedPostCSSVersion && postcss().version.startsWith("7.")) {
+    isCheckedPostCSSVersion = true;
+    const pkg = readPackageJson();
```

Comment:
It will add one more fs call for each `css` file, reduce perf very well

Reply:
As I understand, `fs` call will be only once and only for `postcss 7`

Reply:
We can solve it on two ways:
- webpack already reads `package.json`, need look where we store it
- using `this.fs`, so each call will be cacheable

Reply:
But it is invalid, in monorepos you can have multiple packages with different versions of postcss

Reply:
OK, one minute, I fix it.

Reply:
Perhaps it is better to import the package.the json file at the beginning. So it will be cached: 'const pkg = require(resolve(process. cmd ()," package.json"));`

Your opinion?

Reply:
@alexander-akait can you show examples of such repositories? I'm talking about the mono repository.